Content.Server/Resist/EscapeInventorySystem.cs

```diff
@@ -96,7 +100,14 @@ private void OnEscape(EntityUid uid, CanEscapeInventoryComponent component, Esca
         if (args.Handled || args.Cancelled)
             return;
 
-        if (TryComp<BeingCarriedComponent>(uid, out var carried))
+        if (TryComp<BeingCarriedComponent>(uid, out var carried)) // Start of carrying system of nyanotrasen.
+        {
+            _carryingSystem.DropCarried(carried.Carrier, uid);
+            return;
+        } // End of carrying system of nyanotrasen.
+
+
+        if (TryComp<BeingCarriedComponent>(uid, out carried))
```

Comment:
youve doubled up this guard clause, we only need one. Preferably the better commented one lol